modin/engines/base/io/io.py

```diff
@@ -518,22 +518,29 @@ def read_hdf(
         chunksize=None,
         **kwargs,
     ):  # noqa: PR01
+        from modin.pandas.io import HDFStore
```

Comment:
Using import at the beginning of the file leads to a change in the import order, which in turn breaks `pydocstyle` test.